### PR TITLE
fix: bump tailwind-config to use arbitrary values, replace classes

### PR DESCRIPTION
**Pull request by dine**

### CODE REVIEW

**Review by jpsc — APPROVED** [review]

File: src/components/ui-data-term.vue [hunk 1]
```diff
@@ -1,6 +1,6 @@
 <template>
     <dt
-        class="w-full block text-dark-3 font-normal md:py-3 pt-3 md:border-b border-light-2 md:max-w-1/3 md:flex-1/3"
+        class="w-full block text-dark-3 font-normal md:py-3 pt-3 md:border-b border-light-2 md:flex-[33.333%]"
```
[the remark this answers is not in the document]
Comment (dine) [reply src/components/ui-data-term.vue:3]: It felt useless, the flex declaration already sets the width, right?
I also tried here adding and removing content and it looks the same:
https://funda-frontend.github.io/ui/refs/pull/936/merge/?path=/story/layout-data-list--data-list

**Review by ConnorChristensen — DISMISSED** [review]
Is this actually a breaking change?

**Comment by dine** [comment]
@ConnorChristensen the @funda/tailwind-config is a breaking change and a dependency here, so updating @funda/ui will also bump @funda/tailwind-config in some projects that don't declare it explicitly, right?

**Review by ConnorChristensen — APPROVED** [review]
Thanks for doing this!

**Pull request merged**